charts/jenkins/VALUES_SUMMARY.md

```diff
@@ -99,9 +99,10 @@ The following tables list the configurable parameters of the Jenkins chart and t
 | `controller.resources`                | Resources allocation (Requests and Limits) | `{requests: {cpu: 50m, memory: 256Mi}, limits: {cpu: 2000m, memory: 4096Mi}}`|
 | `controller.initContainerEnv`         | Environment variables for Init Container                                 | Not set |
 | `controller.containerEnv`             | Environment variables for Jenkins Container                              | Not set |
-| `controller.usePodSecurityContext`    | Enable pod security context (must be `true` if `runAsUser` or `fsGroup` are set) | `true` |
+| `controller.usePodSecurityContext`    | Enable pod security context (must be `true` if `runAsUser`, `fsGroup`, or `podSecurityContextOverride` are set) | `true` |
```

Comment:
should we deprecate runAsUser and fsGroup?

Reply:
Add comments along the lines of deprecating them?  Completely pulling them out would be a major version bump.